spec/suites/layer/vector/PolygonSpec.js

```diff
@@ -175,6 +184,25 @@ describe('Polygon', function () {
 			}).to.throwException('Must add layer to map before using getCenter()');
 		});
 
+		it('should compute same center for low and high zoom', function () {
+			var latlngs = [
+				[[0, 0], [0.010, 0], [0.010, 0.010], [0, 0.010]]
+			];
+			var layer = new L.Polygon(latlngs).addTo(map);
+			map.setZoom(0);
+			var center = layer.getCenter();
+			map.setZoom(18);
```

Comment:
Same here, and in all other places.

Reply:
I changed it to `beaforeEach` so for each test the map is new created and the zoom & center will be set